contrib/test/integration/build/conmon.yml

```diff
@@ -4,7 +4,7 @@
   git:
     repo: "https://github.com/containers/conmon.git"
     dest: "{{ ansible_env.GOPATH }}/src/github.com/containers/conmon"
-    version: v2.0.15
+    version: v2.0.20
```

Comment:
This needs a quay image update as well.

Reply:
Will do once we merge https://github.com/cri-o/cri-o/pull/4351. Or should I combine both PRs into one?

Reply:
OK, let me merge this one into #4351